packages/clerk_flutter/lib/src/widgets/user/clerk_user_profile.dart

```diff
@@ -0,0 +1,454 @@
+import 'dart:io';
+
+import 'package:clerk_auth/clerk_auth.dart' as clerk;
+import 'package:clerk_flutter/clerk_flutter.dart';
+import 'package:clerk_flutter/src/assets.dart';
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:phone_input/phone_input_package.dart';
+
+final _emailRE = RegExp(
```

Comment:
Add comment to indicate how this reg-ex works and does it complete with rfc

Reply:
Better still, I'm now using the `email_validator` package. It's another dependency, but I think that's probably okay?